localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_dryad_1;Wilds of Cenarius;;Cenariusdickicht;;;;;;;;;;;x
+ca_culture_dryad_2;Wilds of Cenarius;;Cenariusdickicht;;;;;;;;;;;x
+ca_culture_dryad_3;Wilds of Cenarius;;Cenariusdickicht;;;;;;;;;;;x
+ca_culture_dryad_4;Wilds of Cenarius;;Cenariusdickicht;;;;;;;;;;;x
+ca_culture_dryad_desc;In this sacred grove, the children of Cenarius frolic under the silent, watchful eyes of those who would give their lives to protect these wilds. The keepers of this grove are deadly with spear as well as bow, and they are willing to sacrifice anything against those who would threaten life itself.;;In diesem heiligen Hain tummeln sich die Kinder des Cenarius unter den stillen, wachsamen Augen derer, die ihr Leben f�r den Schutz dieser Wildnis geben w�rden. Die H�ter dieses Hains sind sowohl mit Speer als auch Bogen t�dlich und sie sind bereit, alles gegen jene zu opfern, die das Leben selbst bedrohen w�rden.;;;;;;;;;;;x
+ca_culture_vulpera_1;Vulperan Caravansary;;Vulpera-Karawanserei;;;;;;;;;;;x
+ca_culture_vulpera_2;Vulperan Caravansary;;Vulpera-Karawanserei;;;;;;;;;;;x
+ca_culture_vulpera_3;Vulperan Caravansary;;Vulpera-Karawanserei;;;;;;;;;;;x
+ca_culture_vulpera_4;Vulperan Caravansary;;Vulpera-Karawanserei;;;;;;;;;;;x
+ca_culture_vulpera_desc;The vulperan people normally live a very nomadic and peaceful life within their caravans and burrows. When threatened, however, vulperan marksmen become equally deadly as they are eager to defend their people as well as their treasures.;;Die Vulperaner leben normalerweise ein nomadisches und sehr friedliches Leben in ihren Karawanen und H�hlen. Wenn sie jedoch bedroht werden, werden vulperanische Scharfsch�tzen ebenso t�dlich, wie sie bestrebt sind, ihr Volk und ihre Sch�tze zu verteidigen.;;;;;;;;;;;x
+ca_culture_vrykul_1;Vrykul Drake Grounds;;Vrykul-Drachengel�nde;;;;;;;;;;;x
+ca_culture_vrykul_2;Vrykul Drake Grounds;;Vrykul-Drachengel�nde;;;;;;;;;;;x
+ca_culture_vrykul_3;Vrykul Drake Grounds;;Vrykul-Drachengel�nde;;;;;;;;;;;x
+ca_culture_vrykul_4;Vrykul Drake Grounds;;Vrykul-Drachengel�nde;;;;;;;;;;;x
+ca_culture_vrykul_desc;Having descended from the titans' creations themselves, the vrykul have always shown mastery in physical combat and are renowned for having been the first to domesticate proto-drakes as mounts, succeeding their iron brethren. Vrykul warriors are truly a fearsome sight whether they be upon a proto-drake or charging land-bound with a battle axe in hand.;;Da die Vrykul von den Sch�pfungen der Titanen selbst abstammen, haben sie stets ihre Meisterschaft im k�rperlichen Kampf unter Beweis gestellt und sind daf�r bekannt, dass sie die ersten waren, die Protodrachen als Reittiere domestiziert haben, als Nachfolger ihrer eisernen Br�der. Vrykul-Krieger sind ein wahrhaft furchteinfl��ender Anblick, ob sie nun auf einem Protodrachen sitzen oder mit einer Streitaxt in der Hand am Boden k�mpfen.;;;;;;;;;;;x
```

Comment:
A small change. I'm not sure if this is right, it might be fine the way it is, but `Having descended from the titans'` doesn't sound right to me. I feel like it is missing something. Not sure if my change is any better though, though it does match the precedent set by the Keep of Origination building.
```suggestion
ca_culture_vrykul_desc;Descended from the titans' creations themselves, the vrykul have always shown mastery in physical combat and are renowned for having been the first to domesticate proto-drakes as mounts, succeeding their iron brethren. Vrykul warriors are truly a fearsome sight whether they be upon a proto-drake or charging land-bound with a battle axe in hand.;;Da die Vrykul von den Sch�pfungen der Titanen selbst abstammen, haben sie stets ihre Meisterschaft im k�rperlichen Kampf unter Beweis gestellt und sind daf�r bekannt, dass sie die ersten waren, die Protodrachen als Reittiere domestiziert haben, als Nachfolger ihrer eisernen Br�der. Vrykul-Krieger sind ein wahrhaft furchteinfl��ender Anblick, ob sie nun auf einem Protodrachen sitzen oder mit einer Streitaxt in der Hand am Boden k�mpfen.;;;;;;;;;;;x
```

Reply:
done